config/taxonomy/codegen.spec.yaml

```diff
@@ -1,9 +1,9 @@
 openapi: 3.0.1
 info:
   version: 1.0.0
-  title: Some Swagger
+  title: Policy Manager Specification
   license:
-    name: MIT
+    name: IBM
```

Comment:
Apache 2.0

Reply:
Do we need to change the title to Policy Manager Service ?

Reply:
fixed. Thanks.

Reply:
Changed title to Policy Manager Service